ext/chart_kit/ang/crmChartKit/crmSearchDisplayChartKit.component.js

```diff
@@ -276,11 +276,11 @@
         this.chart
           .width(() => (this.settings.format.width))
           .height(() => (this.settings.format.height))
-          .on('pretransition', chart => {
-            chart.selectAll('text').attr('fill', this.settings.format.labelColor);
+          .on('pretransition', () => {
+            this.chart.selectAll('text').style('fill', this.settings.format.labelColor);
```

Comment:
Are you sure this anonymous function is getting bound to the same `this` as the outer function?

Reply:
Cool.